README.md

```diff
@@ -1,75 +1,43 @@
-# KintoHub {name}-example
+# KintoHub Gatsby Example
 
 ## Overview
-TODO: Cool description / goal of what technology you are bringing onto the plate with what language. This should be sweet and concise.
+Gatsby makes it easy to build websites and apps based on React. Taking a static gastby website and deploying it on KintoHub couldn't be easier!
 
-[Live Example](live/example/on/kintohub)
+[View Live Example](...)
 
 __About KintoHub:__
 
 KintoHub aligns teams to ship & operate cloud native apps with ease. [Learn More](https://www.kintohub.com)
-## [Optional] Requirements
-
-- Ensure you have an account on [KintoHub](www.kintohub.com/sign-up)
-- [Optional] Required Kintohub features: [list features] (like pro sub features)
-- [Optional] specific packages needed for project to run
 
 ## Deployment
-1. Apply this template to your [Github](https://github.com/kintohub/example-template/generate)
-2. Create a [KintoBlockType](www.link.to.create.kintoblock.type.here) and include:
-
-You're now good to build! Click `Build Latest Commit`. Once complete, Click  Now click `Add To Project`.
-
-3. Create Project and include:
+1. Apply this template to your [Github](https://github.com/kintohub/gatsby-example/generate)
+2. Create a [Website KintoBlock](https://beta.kintohub.com/app/dashboard/)
+4. Set the **name** of your kinto block
+5. Choose **Static from build** as your type
+6. Choose `Node.js` as the **language** and `11.7.0` as the **version**
```

Comment:
We have more versions to choose now, 11.7.0 is no longer on the top, better use `lastest` or `13.1.0-alpine`

Reply:
@edwardkcyu never use latest. It causes issues when pods restart and pull new images for builds.

Also choosing X version, it will always change over time. Happy to change it now - but managing 20+ examples to stay at the latest won't be the solution. We'll need to prio some versions on top of others via our db.